ports/stm32/rfcore.c

```diff
@@ -661,9 +665,9 @@ void rfcore_ble_hci_cmd(size_t len, const uint8_t *src) {
     LL_C1_IPCC_SetFlag_CHx(IPCC, ch);
 }
 
-void rfcore_ble_check_msg(int (*cb)(void *, const uint8_t *, size_t), void *env) {
+size_t rfcore_ble_check_msg(void (*cb)(void *, const uint8_t *, size_t), void *env) {
```

Comment:
I know this was already like this, but maybe a good opportunity (while fixing the `void*`) to make this a typedef `rfcore_ble_msg_cb` or something.

Reply:
Had just been thinking that myself after your void* comment above!